central/globaldb/telemetry.go

```diff
@@ -0,0 +1,31 @@
+package globaldb
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/postgres/pgadmin"
+	"github.com/stackrox/rox/pkg/postgres/pgconfig"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+)
+
+// Gather database related information.
+// Current properties we gather:
+// "Database size"
+var Gather phonehome.GatherFunc = func(ctx context.Context) (map[string]any, error) {
+	props := make(map[string]any)
+
+	_, config, err := pgconfig.GetPostgresConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get postgres config")
+	}
+
+	dbSize, err := pgadmin.GetDatabaseSize(config, pgconfig.GetActiveDB())
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get databaze size")
+	}
+
+	props["Database Size"] = dbSize
```

Comment:
Couple of suggestions:
Database version:  We should do this one while we are here because it becomes important for BYODB.  https://github.com/stackrox/stackrox/blob/691bf1ea21a2c01dc20a71c823a38fd2e31cd429/central/globaldb/postgres.go#L164

Do we want the table specific sizes too?
https://github.com/stackrox/stackrox/blob/691bf1ea21a2c01dc20a71c823a38fd2e31cd429/central/globaldb/postgres.go#L178

Reply:
DB version is a good idea, will try to add. But I would like to avoid table specific sizes, at least in the first iteration, because it will significantly increase amount of data sent -- and the added value is not clear, usually we need this for troubleshooting, but in that scenario we already can get this information via diagnostic bundle. How does it sound?

Reply:
Yeah that makes sense.  Here is one more that should be trivial and goes with version.  Since we have customers asking to use external, it may be helpful to track that along with the version.  But I can also see an argument not doing it. 

pgconfig.IsExternalDatabase()